src/openzaak/accounts/migrations/0006_migrate_from_auth_adfs_db.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 3.2.12 on 2022-04-12 09:43
+
+import sys
+
+from django.db import migrations
+
+from django_auth_adfs.config import provider_config, settings as auth_adfs_settings
+from mozilla_django_oidc_db.forms import OpenIDConnectConfigForm
+
+
+def from_auth_adfs_to_mozilla_oidc(apps, schema_editor):
+    ADFSConfig = apps.get_model("django_auth_adfs_db", "ADFSConfig")
+    OpenIDConnectConfig = apps.get_model(
+        "mozilla_django_oidc_db", "OpenIDConnectConfig"
+    )
+
+    adfs_config = ADFSConfig.objects.first()
+    if adfs_config is None:
+        return
+
+    oidc_config = OpenIDConnectConfig.objects.first()
+    if oidc_config and oidc_config.oidc_op_discovery_endpoint:
+        print("Existing OIDC config found, not overwriting it.", file=sys.stderr)
+        return
+    elif oidc_config is None:
+        oidc_config = OpenIDConnectConfig()
+
+    config_url = f"https://{auth_adfs_settings.SERVER}/{auth_adfs_settings.TENANT_ID}/"
+    provider_config.load_config()
+
+    # copy configuration over
+    form = OpenIDConnectConfigForm(
+        instance=oidc_config,
+        data={
+            "enabled": adfs_config.enabled,
+            "oidc_rp_client_id": adfs_config.client_id,
+            "oidc_rp_client_secret": adfs_config.client_secret,
+            "oidc_rp_sign_algo": "RS256",
```

Comment:
Just checking, is this the only supported algorithm for ADFS?

Reply:
not sure, but I was checking the documentation of Open Zaak where it describes that you should enter `RS256`. In django-auth-adfs I saw a couple more, see https://github.com/snok/django-auth-adfs/blob/master/django_auth_adfs/backend.py#L67